### PR TITLE
Add new endpoints

### LINKED ISSUE #31: Add new endpoints: get all bank accounts and sign one of the accounts

### PR DESCRIPTION
**Pull request by matiasgaratortiz**
Closes #31

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -6,6 +6,13 @@
   "rules": {
     "func-names": "off",
     "no-console": "off",
-    "prefer-template": "off"
+    "prefer-template": "off",
+    "node/no-unpublished-require": "off"
```
Comment (fvictorio) [inline .eslintrc]: Maybe we should create a `.eslintrc` in the server tests directory and make it extend from `server/.eslintrc`. Rules like this make no sense in tests, but may be useful in the main code.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,6 @@
 node_modules/
 .idea/
+
+# testing
+coverage/
+.coveralls.yml
```
Comment (fvictorio) [inline .gitignore]: Why is `.coveralls.yml` ignored? It seems like a config file.

File: server/package.json [hunk 3]
```diff
@@ -23,8 +24,30 @@
...
-    "prettier": "^1.12.1"
+    "prettier": "^1.12.1",
+    "supertest": "^3.1.0"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "**/controllers/**/*.js",
+      "**/routes/**/*.js",
+      "**/etc/**/*.js",
+      "**/*.js",
```
Comment (fvictorio) [inline server/package.json]: Does it make sense to whitelist three directories and then just include all `.js` files?

File: .eslintrc (hunk 1 above)
Comment (matiasgaratortiz) [reply .eslintrc]: :+1:

File: .gitignore (hunk 2 above)
Comment (matiasgaratortiz) [reply .gitignore]: mmm it sounds like I copied and pasted that... I'll remove it

File: server/package.json (hunk 3 above)
Comment (matiasgaratortiz) [reply server/package.json]: Yes, I don't remember why include the `**/*.js`

**Pull request merged**